### DISCUSSION TITLE
Demo start time

### DISCUSSION THREAD

**Discussion by Maleandr** [saul/demofile#206] [Questions] [answered]
I need to find in which time demo started e.g. freeze time, round time etc, so that i can start counting my own time. I'm able to do that after receiving round end/round start, but not before.

Is there any property i missed, where is stored time of server, when demo was started?

**Comment by saul** [comment, marked as answer]
What do you mean by "start counting my own time"? Do you need to know how long the demo has been running at a given point?

If so you can multiply DemoFile#currentTick by the tick rate. The tick rate is specified in the svc_ServerInfo packet. 

Does that help?

**Comment by saul** [comment]
I'm going to close this. If you have any more questions feel free to comment here or create a new issue if it's a totally different question.
